### PR TITLE
Data download/cleaning scripts

### PR DESCRIPTION
**Pull request by samzzzzzh**

### CODE REVIEW

File: src/R/data_splitting.R [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  recipes::step_center(recipes::all_predictors()) %>%
+  recipes::step_scale(recipes::all_predictors())
+
+fire_scaled <- fire_recipe %>%
+  prep() %>%
+  bake(fire_train)
+
+write_csv(fire_train, paste0(opt$out_dir1, "/training_data.csv"))
+write_csv(fire_test, paste0(opt$out_dir2, "/testing_data.csv"))
+write_csv(fire_scaled, paste0(opt$out_dir3, "/scaled_and_centered_training_data.csv"))
```
Comment (a-kong) [inline src/R/data_splitting.R:34]: you can just use one command line argument! and update the usage to have one <out_dir>
```suggestion
write_csv(fire_train, paste0(opt$out_dir, "/training_data.csv"))
write_csv(fire_test, paste0(opt$out_dir, "/testing_data.csv"))
write_csv(fire_scaled, paste0(opt$out_dir, "/scaled_and_centered_training_data.csv"))
```

**Review by Jaskaran1116 — APPROVED** [review]
looks good ! nice work sam!

**Review by a-kong — APPROVED** [review]
looks good! also change cleanedData.csv to cleaned_data.csv

File: src/R/data_tidying.R [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+library(tidyverse)
+library(docopt)
+
+opt <- docopt(doc)
+
+data <- read_csv(opt$file_path, show_col_types = FALSE)
+
+data <- select(data, -c('X','Y','month','day'))
+
+write_csv(data, paste0(opt$out_dir, "/cleanedData.csv"))
```
Comment (a-kong) [inline src/R/data_tidying.R:21]: snake case!!
```suggestion
write_csv(data, paste0(opt$out_dir, "/cleaned_data.csv"))
```

**Review by a-kong — APPROVED** [review]
one more thing!

**Pull request merged**